src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -2888,8 +2888,8 @@ public void setStereoscopy(String stereoscopy) {
 		SBSL,
 		SBSR,
 		HSBSL,
-		OUL,
-		OUR,
+		ABL,
+		ABR,
 		HOUL,
```

Comment:
@valib `HOUL` doesn't exist anymore, isn't it ?
https://ffmpeg.org/ffmpeg-filters.html#stereo3d